presto-docs/src/main/sphinx/release/release-0.179.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.179
+=============
+
+General Changes
+---------------
+
+* Fix issue which could cause incorrect results when processing dictionary encoded data. If the expression
```

Comment:
Fix an issue which can cause ...

If the expression fails on bad input, the results from filtered-out rows containing bad input may (or will?) be included *in* the query output.